### PR TITLE
PTI-80: Update keycloak connection strings.

### PR DESCRIPTION
**Pull request by NickPhura**
- Updated some linting issues.
- Updated a few package versions to satisfy some of the dependency Bot issues.
- Updated .gitignore to remove lock files.
- Update mongoose settings to address deprecation warning.

### CODE REVIEW

File: admin/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "keycloak-angular": "5.1.0",
     "leaflet": "1.5.1",
     "linkify-it": "2.2.0",
-    "lodash": "4.17.15",
+    "lodash": "4.17.13",
```
Comment (danieltruong) [inline admin/package.json]: Remove lodash

File: public-lng/package.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
     "leaflet.markercluster": "1.4.1",
     "leaflet": "1.4.0",
     "linkify-it": "2.2.0",
-    "lodash": "4.17.15",
+    "lodash": "4.17.13",
```
Comment (danieltruong) [inline public-lng/package.json]: Remove

File: admin/package.json (hunk 1 above)
Comment (NickPhura) [reply admin/package.json]: This might be better as its own jira task, since lodash is used in the code in about 100 places currently.

Comment (NickPhura) [reply admin/package.json]: Created a ticket: https://bcmines.atlassian.net/browse/PTI-140

**Pull request merged**